src/js/buffer.js

```diff
@@ -48,9 +48,29 @@ function Buffer(subject) {
 
 
 // Buffer.byteLength(string)
-Buffer.byteLength = function(str) {
-  // FIXME: Returns actual byte length of string not counts of characters.
-  return str.length;
+Buffer.byteLength = function(str, enc) {
```

Comment:
As we are currently supporting only 'utf8' for string encoding. that I think we don't need to get additional parameter for distinguishing string encoding. Furthermore other than 'utf8' encoding should cause raising of an exception like `Error: unsupported encoding`.